src/components/WithdrawPage.tsx

```diff
@@ -159,108 +167,101 @@ const WithdrawPage = (props: Props): ReactElement => {
                   )
                 })}
               </RadioGroup>
-
-              {tokensData.map((token, index) => (
-                <div key={index}>
-                  <TokenInput
-                    {...token}
-                    // inputValue={parseFloat(token.inputValue).toFixed(5)}
-                    onChange={(value): void =>
-                      onFormChange({
-                        fieldName: "tokenInputs",
-                        value: value,
-                        tokenSymbol: token.symbol,
-                      })
-                    }
-                    disabled={poolData?.isPaused}
-                  />
-                  {index === tokensData.length - 1 ? (
-                    ""
-                  ) : (
-                    <div className="formSpace"></div>
-                  )}
-                </div>
-              ))}
-              <div className={"transactionInfoContainer"}>
-                <div className="transactionInfo">
-                  <div className="transactionInfoItem">
-                    {reviewData.priceImpact.gte(0) ? (
-                      <span className="bonus">{t("bonus")}: </span>
-                    ) : (
-                      <span className="slippage">{t("priceImpact")}</span>
-                    )}
-                    <span
-                      className={
-                        "value " +
-                        (reviewData.priceImpact.gte(0) ? "bonus" : "slippage")
+              <Stack spacing={3}>
+                {tokensData.map((token, index) => (
+                  <div key={index}>
+                    <TokenInput
```

Comment:
can `key=` attribute go directly to `TokenInput` component as a prop and remove the div?